app/src/main/java/com/jerboa/model/RemoveViewModel.kt

```diff
@@ -0,0 +1,68 @@
+package com.jerboa.model
+
+import android.content.Context
+import android.util.Log
+import android.widget.Toast
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.focus.FocusManager
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.jerboa.R
+import com.jerboa.api.API
+import com.jerboa.api.ApiState
+import com.jerboa.api.toApiState
+import com.jerboa.ui.components.common.apiErrorToast
+import it.vercruysse.lemmyapi.v0x19.datatypes.CommentResponse
+import it.vercruysse.lemmyapi.v0x19.datatypes.CommentView
+import it.vercruysse.lemmyapi.v0x19.datatypes.RemoveComment
+import kotlinx.coroutines.launch
+
+class RemoveViewModel : ViewModel() {
```

Comment:
Debating whether to have this used for both Posts and Comments, or just Comments.

Reply:
If you want it to use both you will probably have to make it into a "helper construct"

Reply:
I'll pry use two different viewModels then.